manifests/g/Google,Inc/GoogleDrive/64.0.4.0/Google,Inc.GoogleDrive.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Google,Inc.GoogleDrive

```

```suggestion
PackageIdentifier: Google.Drive
```
Publisher folder already exists, so new packages should use the same folder